content/docs/deployment/csmoperator/modules/replication.md

```diff
@@ -5,23 +5,47 @@ description: >
   Pre-requisite for Installing Replication via Dell CSM Operator
 ---
 
-The CSM Replication module for supported Dell CSI Drivers can be installed via the Dell CSM Operator. Dell CSM Operator will deploy CSM Replication sidecar and the complemental CSM Replication controller manager.
+The CSM Replication module for supported Dell CSI Drivers can be installed via the Dell CSM Operator. Dell CSM Operator will deploy the CSM Replication sidecar and the CSM Replication Controller Manager.
```

Comment:
Can include replication Crds?

Reply:
The Replication CRDs are not installed via Operator. There is a step that installs the CRDs using repctl. However in the future we may have Operator install all fo the CRDs.